src/bluecadet/core/SettingsManager.cpp

```diff
@@ -163,11 +165,13 @@ void SettingsManager::applyToAppSettings(ci::app::App::Settings * settings) {
 		settings->setMultiTouchEnabled(true);
 	}
 
-	// Keep window top-left within display bounds
-	if (settings->getWindowPos().x == 0 && settings->getWindowPos().y == 0) {
+	// Default window position to centered in display if no custom pos has been set
+	if (mWindowPos == ivec2(0)) {
```

Comment:
@mattfelsen One thing I just realized is that you won't be able to set a window position of 0,0 this way, but I think that's fine for now. Not sure what a clean workaround would be (perhaps a bool or some absurd default value like -999999).

Reply:
Ah good point. Will give it a think :)